src/msonedrive/Module.md

```diff
@@ -0,0 +1,17 @@
+Connects to Microsoft OneDrive and provides information on the files which have been stored on OneDrive.
+
+# Module Overview
+This module contains operations for accessing the items stored in OneDrive.
+
+## Compatibility
+|                     |    Version     |
+|:-------------------:|:--------------:|
+| Ballerina Language  | 1.2.0   |
+| Microsoftgraph REST API | v1.0          |
+
+## Supported Operations
+- Get item from root directory
```

Comment:
```suggestion
- Get an item from the root directory
```

Reply:
Updated the text as follows.
"Get an item from the root directory"